hack/operator-sdk.sh

```
@@ -0,0 +1,37 @@
+set -e
+
+OUTPUT_PATH=${1:-./bin/operator-sdk}
+VERSION=${2:-"v1.39.0"}
+
+GOOS=$(go env GOOS)
+GOARCH=$(go env GOARCH)
+BIN="operator-sdk"
+BIN_ARCH="${BIN}_${GOOS}_${GOARCH}"
+OPERATOR_SDK_DL_URL="https://github.com/operator-framework/operator-sdk/releases/download/${VERSION}"
+
+if [[ "$GOOS" != "linux" && "$GOOS" != "darwin" ]]; then
```

Shellcheck (0.10.0)

[warning] In POSIX sh, [[ ]] is undefined.

(SC3010)